src/binary-reversal/index.js

```diff
@@ -3,6 +3,28 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+  if (typeof value !== "string" || value instanceof String || !value) {
+    return value;
+  }
+  let answer = [];
+  let cal = (a) => {
+    let num = a / 2;
+    let frac = Math.round(num % 1);
+    answer.push(frac);
+    if (num >= 1) {
+      cal(num);
+    }
+  };
+  cal(value);
+
+  let i = answer.length;
+  while (i < 8) {
+    answer.push(0);
+    i++;
+  }
```

Comment:
`padStart` works for padding strings with anything you want up to a certain length.